docs/index.bs

```diff
@@ -1714,6 +1837,7 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/
       [Constructor(DOMString type, FetchEventInit eventInitDict), Exposed=ServiceWorker]
       interface FetchEvent : ExtendableEvent {
         [SameObject] readonly attribute Request request;
+        readonly attribute Promise&lt;Response?&gt; preloadResponse;
```

Comment:
I'm trying to show that `preloadResponse` is a promise that resolves with a response or undefined. What's the right IDL in this case? "any" seems less informative.